### ISSUE TITLE
missing 'feature' in outputField

### ISSUE THREAD

**Issue by Ying456123** [jpmml/jpmml-xgboost#38]
@vruusmann Now I using the jpmml-xgboost-1.3.11 to generate a xgboost pmml model. However, I found there is not a 'feature in outputField which will cause spss cannot load this pmml model successfully. When I use jpmml-xgboost-1.3.2, I can get this field. Could you help to solve this issue? Thanks a lot.
Comparing two version pmml files:
![image](https://user-images.githubusercontent.com/24699537/61857845-7b484600-aef7-11e9-9b06-1f001a7c2c80.png)

Spss load model error:
```
07/25/2019 14:45:25.660|ERROR||||com.spss.ac.acpmml.translator.BaseTranslator.getAttributeForced(BaseTranslator.java:68)|Score error: the attribute feature is required for OutputField
com.spss.ac.accode.ACException: OutputField element missing required feature attribute.
	at com.spss.ac.acpmml.translator.BaseTranslator.getAttributeForced(BaseTranslator.java:71)
	at com.spss.ac.acpmml.translator.PMMLTranslator.handleOutputField(PMMLTranslator.java:3647)
	at com.spss.ac.acpmml.translator.PMMLTranslator.handleOutput(PMMLTranslator.java:3614)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.handleElements(MiningModelTranslator.java:347)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.parseModel(MiningModelTranslator.java:396)
	at com.spss.ac.acpmml.util.ModelFactory.dealValidModel(ModelFactory.java:490)
	at com.spss.ac.acpmml.util.ModelFactory.createModel(ModelFactory.java:447)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.handleSegment(MiningModelTranslator.java:136)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.handleSegmentation(MiningModelTranslator.java:248)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.handleElements(MiningModelTranslator.java:353)
	at com.spss.ac.acpmml.translator.MiningModelTranslator.parseModel(MiningModelTranslator.java:396)
	at com.spss.ac.acpmml.util.ModelFactory.dealValidModel(ModelFactory.java:490)
	at com.spss.ac.acpmml.util.ModelFactory.createModel(ModelFactory.java:447)
	at com.spss.ac.acpmml.util.ModelFactory.createModel(ModelFactory.java:163)
```
Thanks a lot. I will also attache the xgboost model, feature map and the generated pmml file.

[boost_test.zip](https://github.com/jpmml/jpmml-xgboost/files/3430464/boost_test.zip)

**Comment by Ying456123** [comment]
From the pmml official document (http://dmg.org/pmml/v4-3/Output.html), seems like predictedValue means select the raw predicted value. Without this feature, maybe model cannot know which value is prediction?

**Comment by vruusmann** [comment]
Accoring to the PMML specification, the default value of the `OutputField@feature` attribute is `predictedValue`: http://dmg.org/pmml/v4-3/Output.html

Therefore, the following two `OutputField` declarations are functionally identical:

Explicit `OutputField@feature="predictedValue"`:
``` xml
<OutputField name="pred(y)" feature="predictedValue"/>
```

Implicit `OutputField@feature="predictedValue"`:
``` xml
<OutputField name="pred(y)"/>
```

TLDR: This is an SPSS bug. Both JPMML-XGBoost library versions 1.3.2 and 1.3.11 are behaving correctly.
